pom.xml

```diff
@@ -277,6 +251,15 @@
                             <rpt>SCRIPT_STYLE</rpt>
                         </mapping>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <!-- includes common copyright headers for sources -->
+                            <groupId>org.kaazing</groupId>
+                            <artifactId>community.license</artifactId>
+                            <type>jar</type>
+                            <version>${project.version}</version>
```

Comment:
Yes, perfect, and thanks for the example link. I'll make the corresponding change.